### PR TITLE
Xsoar linter sleep

### PR DESCRIPTION
**Pull request by orhovy**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://github.com/demisto/demisto-sdk/pull/762

## Description
removed and ignore sleep statments

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by orhovy** [comment]
validate will fail as docker image of `ActiveMQ`, should not be updated. and `ActiveMQ` it fails on the latest tag for docker image. 
lint will fail as the ignore message is not part of pylints messages as it runs on sdk master and sleep message is not yet merged.

File: Packs/ARIAPacketIntelligence/Integrations/ARIAPacketIntelligence/ARIAPacketIntelligence.py [hunk 1]
```diff
@@ -267,7 +267,6 @@ def _wait_for_trid(self, trid: str) -> bool:
                         return True
                     elif 'FAILURE' in tcl_entry['status']:
                         return False
-            time.sleep(0.1)
```
Comment (bakatzir) [inline Packs/ARIAPacketIntelligence/Integrations/ARIAPacketIntelligence/ARIAPacketIntelligence.py:270]: let's keep this and add ignore code

File: Packs/GoogleChronicleBackstory/Integrations/GoogleChronicleBackstory/GoogleChronicleBackstory.py [hunk 2]
```diff
@@ -160,7 +160,6 @@ def validate_response(client, url, method='GET'):
             raise ValueError('Internal server error occurred, please try again later')
         if raw_response[0].status == 429:
             demisto.debug('API Rate limit exceeded. Retrying in {} seconds...'.format(1))
-            time.sleep(1)
```
Comment (bakatzir) [inline Packs/GoogleChronicleBackstory/Integrations/GoogleChronicleBackstory/GoogleChronicleBackstory.py:163]: we should return an error in this case, but as it is a recent contrib, let's keep it as is. from now on with the linter we will not allow this.

**Comment by orhovy** [comment]
opened a new pr after changing the validation for failing only on sleep statements bigger than 10 sec.
https://github.com/demisto/content/pull/9067

**Pull request closed without merging**